code/game/objects/items/weapons/batons.dm

```diff
@@ -81,10 +85,10 @@
 	// Visuals and sound
 	user.do_attack_animation(target)
 	playsound(target, stun_sound, 75, TRUE, -1)
-	add_attack_logs(user, target, "Stunned with [src]")
+	add_attack_logs(user, target, "Knocked down with [src]")
 	// Hit 'em
 	target.LAssailant = iscarbon(user) ? user : null
-	target.Weaken(stun_time)
+	target.KnockDown(knockdown_duration)
 	on_cooldown = TRUE
 	addtimer(CALLBACK(src, .proc/cooldown_finished), cooldown)
```

Comment:
```suggestion
	addtimer(VARSET_CALLBACK(src, on_cooldown, FALSE), cooldown)
```
and remove the `cooldown_finished()`
(although this is technically not in scope)

Reply:
always forget VARSET_CALLBACK exists